gdal/ogr/ogrsf_frmts/geomedia/ogrgeomediadatasource.cpp

```diff
@@ -140,6 +140,28 @@ int OGRGeomediaDataSource::Open( const char * pszNewName, int bUpdate,
         }
     }
 
+    // check for GAliasTable table
+    bool bFoundGAliasTable = false;
+    {
+        CPLODBCStatement oTableList( &oSession );
+        if( oTableList.GetTables() )
+        {
+            while( oTableList.Fetch() )
+            {
+                const CPLString osTableName = CPLString( oTableList.GetColData(2) );
```

Comment:
looking at ogr/ogrsf_frmts/odbc/ogrodbcdatasource.cpp, I see a null ptr check for the return of .GetColData(2). Assigning a nullptr to a CPLString / std::string is undefined behaviour. So perhaps pass the "" as default value

```suggestion
                const CPLString osTableName( oTableList.GetColData(2, "") );
```